From: Cole Robinson <crobinso@redhat.com>
Date: Mon, 10 Aug 2015 12:35:13 -0400
Subject: [PATCH virt-manager] storage: session path should be
 ~/.local/share/libvirt/images

Not .local/libvirt/images :/ App dirs aren't supposed to be in ~/.local

(cherry picked from commit e22aeac5ae23b03e79e1d18c5fc7c6f2ca8b7a8d)
---
 virtinst/storage.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/virtinst/storage.py b/virtinst/storage.py
index 9e90db8..285acb8 100644
--- a/virtinst/storage.py
+++ b/virtinst/storage.py
@@ -81,7 +81,7 @@ class _StorageObject(XMLBuilder):
 def _get_default_pool_path(conn):
     path = "/var/lib/libvirt/images"
     if conn.is_session_uri():
-        path = os.path.expanduser("~/.local/libvirt/images")
+        path = os.path.expanduser("~/.local/share/libvirt/images")
     return path
 
 
